packages/ui-components/src/components/ContextMenu/ContextMenu.component.tsx

```diff
@@ -66,7 +73,7 @@ export const ContextMenu = ({
       allowedPlacements: ["top-start", "top-end", "bottom-start", "bottom-end"],
     }),
     size({
-      boundary: "viewport",
+      boundary: "viewport" as Boundary,
```

Comment:
Please make a ticket or put it somewhere cause it is a bug(not ours) it was before the type Boundary on size method does not accept value "viewport" according to TypeScript

Reply:
For sure, already done https://github.com/cloudoperators/juno/pull/596#discussion_r1834582553. Thank You!